backend/graph/api/schema/mutations.graphqls

```diff
@@ -26,6 +26,9 @@ type MutationRoot {
     addShowToMyList(showId: ID!): AddToCollectionResult!
 
     removeEntryFromMyList(entryId: UUID!): UserCollection!
+
+    updateUserMetadata(birthData: BirthOptions!, nameData: NameOptions!): Boolean!
+    verifyEmail: Boolean!
```

Comment:
Is this actually `verify` or is it more `sendVerificationEmail`.

Reply:
definitely sendVerificationEmail 🙃